test/unit/controllers/Irc_EntryController.test.js

```diff
@@ -167,7 +184,8 @@ describe('UNIT Irc_EntryController', () => {
         publishUpdate: sinon.stub()
       };
       fake_request = {
-	centres: 'foobar',
+	      centres: 'foobar',
+        heartbeat_recieved: new Date(),
```

Comment:
I don't think you need a `heartbeat_received` here? its what is sent from the irc